canvasapi/account.py

```diff
@@ -1702,14 +1804,66 @@ def update_role(self, role, **kwargs):
 @python_2_unicode_compatible
 class AccountNotification(CanvasObject):
     def __str__(self):  # pragma: no cover
```

Comment:
remove `# pragma: no cover`, and add test (if applicable)

Reply:
Added test.